### PR TITLE
Added ClickUI, useEmptyPassphrase, and try clauses for bridge / webusb

### PR DESCRIPTION
**Pull request by brianddk**
* mem_flashblock.py, mem_read.py, mem_write.py
  * Added ClickUI to remove warning messages on client instantiation.
  * Added try / except for systems with both bridge and webusb devices.
* pwd_reader.py, encfs_aes_getpass.py
  * Emulated `useEmptyPassphrase` so that encrypt / decrypt operations will match those of Trezor Password Manager.

### CODE REVIEW

**Comment by matejcik** [comment]
i'm pretty sure the NoPassphraseUI in `encfs_aes_getpass` is wrong. we never disallowed passphrases for encfs

File: tools/mem_write.py [hunk 1]
```diff
@@ -9,17 +10,28 @@ def main():
...
     # Check whether we found any
     if len(devices) == 0:
         print('No TREZOR found')
         return
 
-    # Use first connected device
-    transport = devices[0]
-    debug_transport = devices[0].find_debug()
+    # Check if its available
+    if not debug_transport:
```
Comment (matejcik) [inline tools/mem_write.py]: is there any case where you don't get an exception from the debug device, but get a falsey value for `find_debug`?

**Comment by brianddk** [comment]
> i'm pretty sure the NoPassphraseUI in `encfs_aes_getpass` is wrong. we never disallowed passphrases for encfs

Sure, I'll back it out.  I just referenced the Trezor Password Manager, but didn't look at the storage provider.  I'll make the change.

File: tools/mem_write.py (hunk 1 above)
Comment (brianddk) [reply tools/mem_write.py]: > is there any case where you don't get an exception from the debug device, but get a falsey value for `find_debug`?

I always either get a valid handle or `TransportException`, but I never get `False`
```
bridge:lib02
webusb:001:1:2
```
Any call to `device['webusb'].find_debug()` raises:
```
TransportException: Debug device not available
```

As long as the enumerator grabs bridge device fist, everything works, but if it happens upon the webusb first, it fails.  This is what happened when I originally made the PR.  I recently noticed that `trezord.exe` had abended, so perhaps this is what I was seeing.

I could look at linux behaviour later, perhaps it is more resilient.

Comment (matejcik) [reply tools/mem_write.py]: oooh I understand what you meant now. No, this is not a good way to do that. These scripts are really old.

Instead do something like this:

```python
def get_debug_device():
	for device in enumerate_devices():
		try:
			return TrezorClientDebugLink(device)
		except Exception:
			pass
	raise Exception("no debuggable device found")

def main():
	# ....
	client = get_debug_device()
	client.debug.memory_write(...)
```

Comment (brianddk) [reply tools/mem_write.py]: Will do... I'll update it in the next day or so.

Thx

Comment (brianddk) [reply tools/mem_write.py]: Does `DebugLink` require debug firmware loaded on the device?  I can't seem to get any `DebugLink` method to run.

I [collected logs](https://gist.github.com/brianddk/a735927014e4024b10edc4efacd01026) for running `py.test trezorlib/tests/device_tests/test_msg_getaddress.py` compared to `py.test trezorlib/tests/device_tests/test_msg_getaddress.py`.  The `py.test` run was failing in DebugLink.

Comment (matejcik) [reply tools/mem_write.py]: yes, debuglink requires a debug firmware.
pytest should be able to run on a debugless firmware if you `export INTERACT=1`

**Comment by matejcik** [comment]
superseded by 2813522b05cef4e0e545a101f8b3559a3183b45b

**Pull request closed without merging**